packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -137,7 +137,10 @@ enum HeroFlightDirection {
 /// is, by default, used to do the transition: when going from route A to route
 /// B, route B's hero's widget is placed over route A's hero's widget. If a
 /// [flightShuttleBuilder] is supplied, its output widget is shown during the
-/// flight transition instead.
+/// flight transition instead. If overriding [flightShuttleBuilder] leads to
```

Comment:
Since you've added the transition of `MediaQuery`, is this new comment still needed?

Reply:
This comment is because I added the transition for the default flightShuttleBuilder but the developer can override the shuttleBuilder so it may be important to let them know that they will need the same behavior in their version of the method if they are having the same issue.

Reply:
I think this line is better added to the documentation of `flightShuttleBuilder`, since it's only for people who overrides it. It might be even better to have an example app for it too (but that's beyond the scope of this PR.)